blaze-core/src/main/scala/org/http4s/blazecore/Http1Stage.scala

```diff
@@ -215,6 +218,8 @@ private[http4s] trait Http1Stage[F[_]] { self: TailStage[ByteBuffer] =>
           }
         go()
       } else cb(End)
+      // TODO (YaSi): I don't it's right
+      F.pure(None)
```

Comment:
I have difficulties understanding the new `F.async` function.
This compiles, but it does not seem right.

Reply:
Use `async_` for a drop in replacement to the old signature

Reply:
https://github.com/http4s/http4s/pull/3894/commits/65216331c4c301102e105680ad66ee831e5cdfba